daemons/controld/controld_remote_ra.c

```diff
@@ -58,18 +58,17 @@ typedef struct remote_ra_cmd_s {
     uint32_t status;
 } remote_ra_cmd_t;
 
-enum remote_migration_status {
-    expect_takeover = 1,
-    takeover_complete,
+enum remote_status {
+    expect_takeover     = (1 << 0),
+    takeover_complete   = (1 << 1),
 };
```

Comment:
FYI expect_takeover/takeover_complete are questionable in a bitmask, since they really are an enumerated condition (i.e. the values are exclusive), but since it only had two possibilities (besides 0) I figured it would be fine as long as we make sure only one is set at a time. If you think that's confusing, feel free to leave them as a separate enum.

You might want to define clear/set macros for the bitmask since the trace log can be helpful, but not terribly important in this case

Reply:
I think the clear/set macros make it a lot more obvious what is happening.  Added.